content/blog/2022/2022-12-26-lc0-release-v0290.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Lc0 release v0.29.0"
+# slug: "blog-post-title"
+published: 2022-12-26
+#draft: true
+author: "borg"
+# image: "name"
+# cover: "cover.png" # Default
+---
+
+After a full year without a Lc0 release, [v0.29.0](https://github.com/LeelaChessZero/lc0/releases/tag/v0.29.0) was finally released a few weeks ago, so this is a perfect opportunity to make a blog post about it and explain why it took so long. Moreover, this way we'll get at least one blog post in 2022...
+<!--more-->
+
+In the following list I have broken down and rearranged the bullet list from the release announcement in several categories with more detailed descriptions.
+
+### Attention policy support.
```

Comment:
I think it makes sense to add more context what "attention policy" is, as the first first paragraph/sentence, even though it's kind of explained later.

From the name/description, it sounds like it's a new feature that users can use, rather than internal NN change, not really visible to users (except the engine strength).

See for example http://talkchess.com/forum3/viewtopic.php?f=2&t=81146#p939595 with the confusion.